app/src/main/res/layout-sw600dp/option_story_text_size.xml

```diff
@@ -17,7 +17,7 @@
     android:id="@+id/reading_text_size_item_layout"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"
```

Comment:
I think this condition is important so as it show the divider which is not visible in your screenshots now.